extensions/authentication-jwt/src/services/user.service.ts

```diff
@@ -8,9 +8,10 @@ import {repository} from '@loopback/repository';
 import {HttpErrors} from '@loopback/rest';
 import {securityId, UserProfile} from '@loopback/security';
 import {compare} from 'bcryptjs';
-import {User} from '../models';
 import {UserRepository} from '../repositories';
-
+/* eslint-disable*/
+import {User, UserRelations} from '../models';
+/* eslint-enable */
```

Comment:
For single-line disable, we can use `// eslint-disable-next-line no-unused-vars`